content/en/docs/how-to/mint-your-cw20-token.md

```diff
@@ -25,13 +25,13 @@ There are some advantages below:
 
 The standard CW20 token is already stored in Fetch.ai network.\
 The code is wrapped based on cw20 [v0.13.2](https://docs.rs/cw20/0.13.2/cw20/index.html) \
-Please check [here]({{< relref "/docs/contract-resources/contract-addresses" >}}) for the more addresses.
+Please check [here]({{< relref "/docs/contract-resources/contract-addresses" >}}) for the token factory addresses.
```

Comment:
This line guides cw20’s code id, not a factory address. plz leave is as is or update to the token code id.